library/Ivoz/Provider/Domain/Service/BillableCall/CsvExporter.php

```diff
@@ -48,13 +50,15 @@ public function __construct(
      * @param \DateTime $outDate
      * @param CompanyInterface|null $company
      * @param BrandInterface|null $brand
+     * @param string | null $direction
      * @return string
      */
     public function execute(
         \DateTime $inDate,
         \DateTime $outDate,
         CompanyInterface $company = null,
-        BrandInterface $brand = null
+        BrandInterface $brand = null,
+        $direction = 'outbound'
```

Comment:
Could we use BillableCallInterface::DIRECTION_OUTBOUND here?

Reply:
Varchar columns are variable-length strings, I would not worry about this